Automattic/Simplecons
# Align SVG Code

This is a PR to align all of the SVG code in the icon set.

In this PR I am removing all of the unnecessary `g` group tags and aligning the `rect` code within each icon which defines the canvas size of each icon.

### To Test
- Verify that no icons contain `g` tags.
- Verify that all 24px icons now use the same code for the `rect`.
- Verify that all 16px icons now use the same code for the `rect`.
- Verify that all icons render correctly.
